test/e2e/common/node/lifecycle_hook.go

```diff
@@ -114,7 +124,7 @@ var _ = SIGDescribe("Container Lifecycle Hook", func() {
 		/*
 			Release: v1.9
 			Testname: Pod Lifecycle, post start http hook
-			Description: When a post start handler is specified in the container lifecycle using a HttpGet action, then the handler MUST be invoked after the start of the container. A server pod is created that will serve http requests, create a second pod with a container lifecycle specifying a post start that invokes the server pod to validate that the post start is executed.
+			Description: When a post start handler is specified in the container lifecycle using a HttpGet action, then the handler MUST be invoked after the start of the container. A server pod is created that will serve http requests, create a second pod on the same node with a container lifecycle specifying a post start that invokes the server pod to validate that the post start is executed.
```

Comment:
the description was updated but the code wasn't modified to schedule the pods on the same host